yt/data_objects/tests/test_points.py

```diff
@@ -62,7 +62,7 @@ def test_fast_find_field_values_at_points():
     # sample 100 particles
     nparticles = 100
     ppos = ad["all", "particle_position"]
-    ppos = ppos[np.random.random_integers(0, len(ppos) - 1, size=nparticles)]
+    ppos = ppos[np.random.randint(low=0, high=len(ppos), size=nparticles)]
```

Comment:
Just verifying: `high` is not inclusive, right?

Reply:
yes, that's the important differences between the two apis, and it's advertised in the deprecation warning itself :)